projects/plugins/jetpack/extensions/blocks/videopress/editor.js

```diff
@@ -64,6 +66,37 @@ const videoPressNoPlanMediaPlaceholder = createHigherOrderComponent(
 	'videoPressNoPlanMediaPlaceholder'
 );
 
+const videoPressMediaPlaceholder = createHigherOrderComponent(
+	OriginalPlaceholder => props => {
+		const { name } = useBlockEditContext();
+		if ( name !== 'core/video' ) {
+			return <OriginalPlaceholder { ...props } />;
+		}
+
+		// We will handle video uploads
+		props.handleUpload = false;
```

Comment:
If `define( 'SCRIPT_DEBUG', true );` is defined, then modifying props like this directly will throw `Uncaught TypeError: Cannot add property handleUpload, object is not extensible` and make the block unusable.